TheWarWithin/PriestShadow.lua

```diff
@@ -577,7 +573,10 @@ spec:RegisterAuras( {
     },
     entropic_rift = {
         duration = 8,
-        max_stack = 1
+        max_stack = 1,
+        alias = { "voidheart" },
+        aliasMode = "first",
+        aliasType = "buff"
```

Comment:
This doesn't need to be an alias -- which is when any of multiple buffs can be counted in place of the named buffs.  Remove `entropic_rift` entirely here and under `voidheart`, just add `copy = "entropic_rift"`.  That will make Voidheart and Entropic Rift act as the same buff at all times.